wgpu-hal/src/metal/device.rs

```diff
@@ -75,11 +75,27 @@ impl super::Device {
             },
         };
 
+        let mut temp_options;
+        let options = if !stage.module.runtime_checks {
+            temp_options = layout.naga_options.clone();
+            if !stage.module.runtime_checks {
```

Comment:
We already checked that `!stage.module.runtime_checks`

Reply:
Thanks @teoxoy my mistake